narwhal/node/src/lib.rs

```diff
@@ -247,7 +251,8 @@ impl Node {
         tx_validator: impl TransactionValidator,
         // The prometheus metrics Registry
         registry: &Registry,
-    ) -> Vec<JoinHandle<()>> {
+    ) -> (Vec<JoinHandle<()>>, broadcast::Sender<()>) {
+        let (tx_shutdown, _tr_shutdown) = broadcast::channel(1);
```

Comment:
I am not against adding this channel here, but keep in mind that we'll eventually introduce the `PrimeLifecycleManager` & `WorkerLifecycleManager` https://mysten.atlassian.net/browse/SUI-971 which will be responsible for the nodes' lifecycle - where more general refactoring will be needed. That being said, could we just leave the use of the `reconfiguration` channel as is to propagate the shutdown signal refactor the approach as part of the https://mysten.atlassian.net/browse/SUI-971  ?